addons/resource.language.en_gb/resources/strings.po

```diff
@@ -22486,16 +22486,19 @@ msgctxt "#39153"
 msgid "Windowing system:"
 msgstr ""
 
+#. Label for section of settings
 #: system/settings/settings.xml
 msgctxt "#39154"
 msgid "Movie set rating"
 msgstr ""
 
+#. Value of setting with label #39154 "Movie set rating"
 #: system/settings/settings.xml
 msgctxt "#39155"
 msgid "Average"
 msgstr ""
 
+#. Value of setting with label #39154 "Movie set rating"
```

Comment:
Please remove the reference to another string id. Instead, explain what the setting value does. Otherwise, the explanation will be of no value for the translators.